index.bs

```diff
@@ -902,89 +1068,6 @@ spec: workers; urlPrefix: https://www.w3.org/TR/workers/#
 <section>
```

Comment:
I'd go ahead and remove the entire Display Capabilities section, including the 'onchange' event. This was proposed as a way for us to know when the window moved from an SDR to HDR display (event listener might re-query for HDR properties like luminance and switch to HDR content). But those properties are all now removed. If the media queries path discussed in #135 pans out, we'll instead use the 'changed' event (https://developer.mozilla.org/en-US/docs/Web/API/MediaQueryList/onchange).